lib/Chunk.js

```diff
@@ -417,11 +407,27 @@ class Chunk {
 	}
 }
 
+// TODO remove in webpack 5
+Object.defineProperty(Chunk.prototype, "forEachModule", {
+	configurable: false,
+	value: util.deprecate(function(fn) {
+		this._modules.forEach(fn);
+	}, "Chunk.forEachModule: Use chunk.modulesIterable.forEach(fn) instead")
```

Comment:
Use for(const module of chunk.modulesIterable) instead

An iterable has no forEach method. Here a Set is returned but you can't rely on that. The official return type is Iterable.

Reply:
I’ll update the wording.